src/ios/Common/OSFANLError.swift

```diff
@@ -0,0 +1,45 @@
+enum OSFANLError: Error {
```

Comment:
Just a comment / suggestion. Should we use `OSFBAN` or `OSFBAL` instead? Disregard this comment in case this has already been aligned with the UI/UX team.

Reply:
After [addressing this with the UI/UX team](https://outsystems.slack.com/archives/G0151EGF5V3/p1702920119550089?thread_ts=1702912756.828249&cid=G0151EGF5V3), it was decided that we will follow the `OSFANL` convention. Thanks for picking this up.

Reply:
No problem, thanks for letting me know.